src/block.rs

```diff
@@ -372,17 +372,29 @@ impl Block {
     }
 
     /// Get the size of the block
+    #[deprecated(since = "0.19.1", note = "Please use `Block::size` instead.")]
```

Comment:
This assumes that the next release is going to be `0.19.1`, which might not be the case.